fs/zbd_zenfs.h

```diff
@@ -79,6 +81,8 @@ class ZonedBlockDevice {
   std::atomic<long> active_io_zones_;
   std::atomic<long> open_io_zones_;
   std::condition_variable zone_resources_;
+  std::mutex zone_close_mutex_;
+  boost::optional<IOStatus> zone_close_error_;
```

Comment:
Why do we need optional here? why not just set it to ok?

Reply:
I think the intention of the code carries more clearly when using an option.

Reply:
We want the status to be ok per default